AMBARI-22894-trunk.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/ambari-server/src/main/resources/stack-hooks/before-START/scripts/params.py b/ambari-server/src/main/resources/stack-hooks/before-START/scripts/params.py
```

Comment:
Please do not add this patch file.  (You can remove it in an another commit, no need for a new PR.)